src/Components/Pages/dashboard.module.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(67, 105, 176);
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  background: rgb(67, 105, 177);
+  justify-content: space-between;
+  min-height: 4rem;
+  padding: 0.5rem;
+  color: white;
+}
+
+.brand {
+  margin-left: 40px;
+}
+
+.brand > h1 {
+  font-size: 20px;
+}
+
+.stocklist {
```

Comment:
- [ ] You have done well to make two columns for the cards in mobile view, but the very first card take the entire row. Kindly fix this issue. `Tesla, Inc` should not take the first row alone.
**See screenshot below**
![Screen Shot 2023-02-07 at 10 08 08 PM](https://user-images.githubusercontent.com/62849872/217449423-70b51b1d-5d40-4226-a66d-90097eb09876.png)